userguide/subscription/includes/payment.adoc

```diff
@@ -1,37 +1,37 @@
 
 === Overview
 
-Kill Bill provides a core payment system, that is used in the subscription billing use case to issue payment requests against the generated invoices.
+Kill Bill provides a core payment system that is used in subscription billing to issue payment requests against the generated invoices.
 
-In order to accomodate for that use case, the payment logic has been split into two pieces:
+To accomodate that use case, the payment logic has been split into two pieces:
 
-First, there is a core payment system whose role is to manage the payment states, and route the operation to the correct payment plugin, which in turn interracts with the third party payment gateway. 
-This core payment system is not specific to the subscription billing use case, and can actually be used as a standalone system to provide a layer in front of various payment gateways.
+First, there is a core payment system whose role is to manage the payment states and route the operation to the correct payment plugin, which in turn interracts with the third party payment gateway. 
+This core payment system is not specific to the subscription billing use case. It can be used also as a standalone system to provide a layer in front of various payment gateways.
 For an overview of the payment system in Kill Bill, please check http://docs.killbill.io/latest/userguide_payment.html[the payment guide].
 
-In order to support the subscription billing use case, and issue payments against existing invoices, there is a payment control plugin embedded into Kill Bill core -- see http://docs.killbill.io/latest/payment_control_plugin.html[payment control plugin guide]-- whose role is to provide the necessary logic to map the payments with the invoice:
+In order to support the subscription billing use case, and issue payments against existing invoices, there is a payment control plugin embedded into the Kill Bill core (see http://docs.killbill.io/latest/payment_control_plugin.html[payment control plugin guide]). The role of this plugin is to provide the necessary logic to coordinate the payments with the invoice:
```

Comment:
Maybe change:

> The role of this plugin is to provide the necessary logic to coordinate the payments with the invoice:

The role of this plugin is to provide the necessary logic to coordinate the payments associated with an invoice: